src/pages/admin/crud/VulcanUser.tsx

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import Router from "next/router";
+import { Button, FormControl, InputLabel, MenuItem, Select } from "@material-ui/core";
+
+import { User } from "~/models/user";
+import { useUser } from "~/components/user/hooks";
+import {
+    useCreate,
+    useMulti,
+    useUpdate,
+    useDelete,
+} from "@vulcanjs/react-hooks";
+
+
+export default function UserCrudPage() {
+    // Auth
+    const user = useUser({ redirectTo: "/login" });
+
+    // Create
+    /* 
+    
+    Before creating a user with useCreate, we need to modify the password fields. That's what is done by the vulcan-next API, so we use the it here.
+    You can see below this page an example with useCreate that will throw because of the password field.
+
+    */
+    async function handleCreateSubmit(e) {
+        e.preventDefault();
+
+        const body = {
+            email: e.currentTarget.email.value,
+            password: e.currentTarget.password.value,
+            isAdmin: e.currentTarget.isAdmin.checked || null,
+            groups: e.currentTarget.groups.value || null,
+        };
+
+        try {
+            const res = await fetch("/api/signup", {
```

Comment:
Nope you should use useUpdate instead.
Here you are signin up a user => that's different from creating them.

We need to discuss how to structure that.